### ISSUE TITLE
Medium theme "demo" links to porn related site

### ISSUE THREAD

**Issue by Eggeriley** [getgrav/grav#1452]
The theme Medium by matthew blode if you click on "demo" takes you to a porn site
![screen shot 2017-04-27 at 10 39 56 am](https://cloud.githubusercontent.com/assets/19999634/25496519/1cf97336-2b36-11e7-9a33-2328645851e5.png)
![screen shot 2017-04-27 at 10 42 21 am](https://cloud.githubusercontent.com/assets/19999634/25496556/398773f4-2b36-11e7-826c-65ed055c9418.png)

**Comment by rhukster** [comment]
Eeek :( . We'll remove the demo link.  I assume the github repo i still good?

**Comment by Eggeriley** [comment]
Yes i looked at the github repo and it seems to be fine.

**Comment by Eggeriley** [comment]
you might also want to check under theme "Gravi-k"
the demo link is hacked or something, it takes you to a new page every time.
![screen shot 2017-04-27 at 10 48 05 am](https://cloud.githubusercontent.com/assets/19999634/25496790/1e4ff164-2b37-11e7-9e15-d6019e2d14c9.png)

**Comment by flaviocopes** [comment]
Thanks, I alerted the theme authors

**Comment by mblode** [comment]
Sorry about this. The demo site was linked to a domain name I used to own but let expire. Thank you @flaviocopes for creating that pull request. And thank you @Eggeriley for bringing this to my attention. Sorry for any inconvenience. I hope you enjoy the theme nonetheless. I have moved on to Craft CMS development and therefore I am not working on the theme anymore but you are free to create pull requests.
